vector/v.surf.rst/test_suite/profile_cv.sh

```diff
@@ -0,0 +1,21 @@
+# nc2008/grass7 mapset
+
+OUTFILE=callgrind.out.1196
+
+g.copy vect=elev_lid792_bepts,lidar_test
+v.build lidar_test
+g.region n=221230 s=220816 w=639090 e=639530 res=0.5 -p
+CMD="v.surf.rst in=lidar_test cvdev=cv_test -c --overwrite"
+time $CMD
+
+valgrind --tool=callgrind --callgrind-out-file=$OUTFILE --trace-children=yes $CMD
+
+# http://kcachegrind.sourceforge.net
+kcachegrind $OUTFILE
```

Comment:
Does this mean valgrind and kcachegrind is needed just to run the tests, in CI or others? Does it work on macOS and Windows?

Reply:
I removed [vector/v.surf.rst/test_suite/profile_cv.sh](https://github.com/OSGeo/grass/pull/3590/files/e49e73eb93639ecda321cab92c92ff9aa88e0e3e#diff-3a590f19bacbec9faefc9dda34dbbb6ea3963487cb9ef6537184fc6dffcb87e1) and [vector/v.surf.rst/test_suite/profile_ompcv.sh](https://github.com/OSGeo/grass/pull/3590/files/e49e73eb93639ecda321cab92c92ff9aa88e0e3e#diff-731c175ba9c7b535713b4124c5d920712aacd05598da96c8c755e273fb7c7a39). The unit tests are mainly done by the Python script in vector/v.surf.rst/testsuite. I just used these files to test the speedups and efficiencies.